fontbe/src/features.rs

```diff
@@ -73,19 +73,26 @@ impl FeatureWork {
     }
 
     fn compile(&self, features: &Features, glyph_order: GlyphMap) -> Result<Compilation, Error> {
-        let root_path = if let Features::File(file) = features {
-            OsString::from(file)
-        } else {
-            OsString::new()
+        let compiler = match features {
+            Features::File {
+                fea_file,
+                include_dir,
+            } => {
+                let mut compiler = Compiler::new(OsString::from(fea_file), &glyph_order);
+                if let Some(include_dir) = include_dir {
+                    compiler = compiler.with_project_root(include_dir)
+                }
+                compiler
+            }
+            Features::Memory(fea_content) => {
+                let root = OsString::new();
+                Compiler::new(root.clone(), &glyph_order).with_resolver(InMemoryResolver {
```

Comment:
`include_dir` isn't used for in-memory compilation, does it mean we only properly support resolving external .fea includes when emitting IR and thus compiling features from a file?

Reply:
I'll double-check and add a test; it should work.

Reply:
(in memory should be when the source is in memory, not just when emit ir is off)

Reply:
oh, like when compiling off .glyphs source (which embeds a features string in its single file source) as opposed to designspace+UFO where features.fea are distinct files?